src/main/java/uk/gov/ons/census/fwmtadapter/messaging/CaseUpdatedReceiver.java

```diff
@@ -79,7 +80,11 @@ private void buildAndSendActionInstruction(
     actionInstruction.setSurveyName(caze.getSurvey());
     actionInstruction.setTownName(caze.getAddress().getTownName());
     actionInstruction.setUprn(caze.getAddress().getUprn());
-    actionInstruction.setUndeliveredAsAddress(caze.getUndeliveredAsAddressed());
+
+    if (event.getPayload().getMetadata().getCauseEventType()
```

Comment:
this will cause a null pointer exception if the metadata is null... can we guarantee that there'll always be metadata on the event?